Bottom: a44ab6c4812ce484728b20b3f304c197d8f43a6d
Top:    d2f94d46009ba7c081cc8837e8ab1480f6756fd0
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-02-09 11:07:01 -0600

Fix printf in docker_build

Printf statement was expanding a variable into the format string,
which caused problems when that variable contained patterns that
appeared to be invalid format specifiers.


---

diff --git a/ansible-playbook/library/docker_build b/ansible-playbook/library/docker_build
index 8cc11ed..da0c78c 100644
--- a/ansible-playbook/library/docker_build
+++ b/ansible-playbook/library/docker_build
@@ -42,7 +42,7 @@ build_result="${?}"
 
 if [[ ${build_result} != "0" ]]; then
     # build failed
-    printf "{\"failed\": true, \"changed\": true, \"stdout\": \"${build_log}\"}"
+    printf "{\"failed\": true, \"changed\": true, \"stdout\": \"%s\"}" "${build_log}"
     exit 1
 fi
